components/eamxx/src/share/atm_process/atmosphere_process.cpp

```diff
@@ -93,8 +93,39 @@ void AtmosphereProcess::run (const double dt) {
       compute_column_conservation_checks_data(dt_sub);
     }
 
+    // Init single step tendencies (if any) with current value of output field
+    if (m_compute_proc_tendencies) {
+      start_timer(m_timer_prefix + this->name() + "::compute_tendencies");
+      for (auto& it : m_proc_tendencies_single_step) {
+        const auto& tname = it.first;
+        const auto& fname = m_tend_to_field.at(tname);
+        const auto& f     = get_field_out(fname);
+
+        auto& step_tend = it.second;
+        step_tend.deep_copy(f);
+      }
+      stop_timer(m_timer_prefix + this->name() + "::compute_tendencies");
+    }
+
     run_impl(dt_sub);
 
+    // Complete tendency calculations (if any)
+    if (m_compute_proc_tendencies) {
+      start_timer(m_timer_prefix + this->name() + "::compute_tendencies");
+      for (auto it : m_proc_tendencies_accum) {
+        const auto& tname = it.first;
+        const auto& fname = m_tend_to_field.at(tname);
+        const auto& f     = get_field_out(fname);
+
+        auto& step_tend = m_proc_tendencies_single_step.at(tname);
+        auto& tend      = it.second;
+
+        step_tend.update(f,1/dt_sub,-1/dt_sub);
+        tend.update(step_tend,1.0,1.0);
+      }
+      stop_timer(m_timer_prefix + this->name() + "::compute_tendencies");
+    }
```

Comment:
I wonder if we even need it for those. I guess it just adds code since that's the only place it's called. 

At some point soonish I'll update the conservation checker and if I find something nicer to do for all of them I'll change it. So no, I don't feel strongly.

Reply:
Ok, done.